Set custom VALID_ARCHS to remove support of armv7(s)

michaelloo opened the pull request:
armv7(s) architectures are now unnecessary as we dropped support for iOS 10 devices

rea-myles-abbott commented on MMPixelSDK.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Was this additional file required?
`Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace’s shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace.`

We may not need it.  But it's not a deal breaker.

michaelloo replied:
Yeah it was

rea-myles-abbott reviewed (approved):
👍

Commit:
Set custom VALID_ARCHS to remove support of armv7(s)

armv7(s) architectures are now unnecessary as we dropped support for iOS 10 devices
